src/code/engine_library/calculateBodySurfaceArea.m

```diff
@@ -13,12 +13,12 @@
 
 
 switch WSettings.BSA_calculationmethode
-
+    
     case 'DuBois'        
-        BSA = (0.007184.*(height.*10).^0.725 .*weight.^0.425).*100;
+        BSA = (0.007184.*(height.*10).^0.725 .*weight.^0.425);
         
     case 'Haycock'
-        BSA = (0.024265.* (height.*10).^0.3964.* weight.^0.5378).*100;
+        BSA = (0.024265.* (height.*10).^0.3964.* weight.^0.5378);
     otherwise
         
```

Comment:
In PK-Sim, **DuBois** and **_Mosteller_** methods are implemented.
Not sure where `WSettings.BSA_calculationmethode` is set, but if it comes from PK-Sim it would be on both

Reply:
and this should read `method` not `methode`

Reply:
Haycock not implemented in PK-Sim

Reply:
The Default method of the Reporting engine is DuBois.
methode was changed to method.